src/components/common/LastPost/LastPost.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
```

Comment:
pusta linie jest tutaj niepotrzebna

Reply:
Przy importach częstą praktyką w projektach jest oddzielanie grupy importów pustą linią dla zachowania porządku, więc moim zdaniem nie ma tu błędu.

Reply:
Masz na myśli oddzielanie importów ze względu na to jaki typ danych ze sobą noszą? np propsy osobno, style osobna itd?

Reply:
Tak, to miało być dla zachowania porządku w importach. Maciek zobacz w innych plikach projektu szczególnie tych, które były na starcie.

Reply:
To zależy od preferencji, zwyczajnie ma to być poukładane w logiczny i uporządkowany sposób, puste linie są tu jak najbardziej ok.

Reply:
Ania, widziałem te pliki na starcie :) Pliki na starcie nie są dla mnie odnośnikiem poprawności końcowej. Przeglądam kod i jeśli mam jakieś uwagi to je piszę. Moja uwaga nie musi oznaczać błędu, zauważyłem niespójność to napisałem. Tomek dziękuję za ogromną wiedzę, jak zwykle zresztą.